src/systems/filecoin_blockchain/storage_power_consensus/storage_power_consensus_subsystem.id

```diff
@@ -23,10 +25,14 @@ type StoragePowerConsensusSubsystem struct {//(@mutable)
     validateElectionProof(
         height         block.ChainEpoch
         electionProof  block.ElectionProof
-        workerAddr     addr.Address
+        minerAddr      addr.Address
     ) bool
 
-    validateTicket(tix block.Ticket, pk filcrypto.VRFPublicKey) bool
+    validateTicket(
+        tix        block.Ticket
+        pk         filcrypto.VRFPublicKey
+        minerAddr  addr.Address
```

Comment:
Ditto my comment about this being a worker address if you expect to validate a signature.

Reply:
So in this case it's not about a sig but rather about the fact that new tickets use a minerAddress as part of randomness generation. The idea is that if different miner actors use the same worker keys, we would want them to generate different randomness, so we hash along with the minerAddr. No sig validation here.